clients/client-s3/src/runtimeConfig.browser.ts

```diff
@@ -40,7 +44,9 @@ export const getRuntimeConfig = (config: S3ClientConfig) => {
     maxAttempts: config?.maxAttempts ?? DEFAULT_MAX_ATTEMPTS,
     md5: config?.md5 ?? Md5,
     region: config?.region ?? invalidProvider("Region is missing"),
+    requestChecksumCalculation: config?.requestChecksumCalculation ?? DEFAULT_REQUEST_CHECKSUM_CALCULATION,
     requestHandler: RequestHandler.create(config?.requestHandler ?? defaultConfigProvider),
+    responseChecksumValidation: config?.responseChecksumValidation ?? DEFAULT_RESPONSE_CHECKSUM_VALIDATION,
```

Comment:
I think the browser fields can be omitted because the resolver applies the default values

Reply:
Removed in [`e6da14c` (#6471)](https://github.com/aws/aws-sdk-js-v3/pull/6471/commits/e6da14c41398bc3e146517e99596652925589e08)